django/db/migrations/state.py

```diff
@@ -41,13 +41,21 @@ def get_related_models_recursive(model):
     its superclass through the implicit *_ptr OneToOneField on the subclass.
     """
     def _related_models(m):
-        return [
-            f.related_model for f in m._meta.get_fields(include_parents=True, include_hidden=True)
+        related_fields = [
```

Comment:
Are you talking about the implicit `any(f.model is m for f in related_fields)` below?